demisto_sdk/main.py

```diff
@@ -1,11 +1,241 @@
+)
+@click.option(
+    '--demisto-mock', '-d',
+    help="Add an import for demisto mock, true by default",
+    type=click.Choice(["True", "False"]),
+    default=True
+)
+@click.option(
+    '--common-server', '-c',
+    help="Add an import for CommonServerPython."
+         "If not specified will import unless this is CommonServerPython",
+    type=click.Choice(["True", "False"]),
+    default='False'
+)
+@pass_config
+def extract_code(config, **kwargs):
+    extractor = Extractor(configuration=config.configuration, **kwargs)
+    return extractor.extract_code(kwargs['outfile'])
+
+
+@main.command(name="unify",
+              short_help='Unify code, image and description files to a single Demisto yaml file.')
+@click.option(
+    "-i", "--indir", help="The path to the files to unify", required=True
+)
+@click.option(
+    "-o", "--outdir", help="The output dir to write the unified yml to", required=True
+)
+def unify(**kwargs):
+    unifier = Unifier(**kwargs)
+    return unifier.merge_script_package_to_yml()
+
+
+@main.command(name="validate",
+              short_help='Validate your content files.')
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    '-j', '--conf-json', is_flag=True,
+    default=False, help='Validate the conf.json file.')
+@click.option(
+    '-i', '--id-set', is_flag=True,
+    default=False, help='Create the id_set.json file.')
+@click.option(
+    '-p', '--prev-ver', help='Previous branch or SHA1 commit to run checks against.')
+@click.option(
+    '-c', '--circle', type=click.Choice(["True", "False"]), default='False',
+    help='Is CircleCi or not')
+@click.option(
+    '-b', '--backward-comp', type=click.Choice(["True", "False"]), default='True',
+    help='To check backward compatibility.')
+@click.option(
+    '-g', '--use-git', is_flag=True,
+    default=False, help='Validate changes using git - this will check your branch changes and will run only on them.')
+@pass_config
+def validate(config, **kwargs):
+    sys.path.append(config.configuration.env_dir)
+
+    validator = FilesValidator(configuration=config.configuration, is_backward_check=str2bool(kwargs['backward_comp']),
+                               is_circle=str2bool(kwargs['circle']), prev_ver=kwargs['prev_ver'],
+                               validate_conf_json=kwargs['conf_json'], use_git=kwargs['use_git'])
+    return validator.run()
+
+
+@main.command(name="create",
+              short_help='Create content artifacts.')
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    '-a', '--artifacts_path', help='The path of the directory in which you want to save the created content artifacts')
+@click.option(
+    '-p', '--preserve_bundles', is_flag=True, default=False,
+    help='Flag for if you\'d like to keep the bundles created in the process of making the content artifacts')
+def create(**kwargs):
+    content_creator = ContentCreator(**kwargs)
+    content_creator.run()
```

Comment:
should this return a value? (same as rest of the commands)

Reply:
not really